judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/QueryTest.java

```diff
@@ -101,18 +99,6 @@ public void testLimit() {
         assertEquals(1, list.size());
     }
 
-    @Disabled
-    public void testGetByIdThrowsForNonExistingElement() {
-        myEntityWithOptionalFieldsDao.delete(entity1);
-
-        IllegalArgumentException thrown = assertThrows(
-                IllegalArgumentException.class,
-                () -> myEntityWithOptionalFieldsDao.getById(entity1.get__identifier())
-        );
-
-        assertNotNull(thrown);
-    }
-
```

Comment:
getById returns Optional object if entity is not found therefore it should not throw any exception.